src/status_im/i18n_resources.cljs

```diff
@@ -25,6 +26,7 @@
    :it         (js/require "../translations/it.json")
    :ko         (js/require "../translations/ko.json")
    :ru         (js/require "../translations/ru.json")
+   :tr         (js/require "../translations/tr.json")
```

Comment:
This `../` is in relation to what? @yenda maybe you can help us out. Tests are failing here on importing this file.

Reply:
Translations also need to be added here for tests https://github.com/status-im/status-react/blob/develop/src/mocks/js_dependencies.cljs#L194

Reply:
@jinhojang6

Reply:
Should we document this somewhere? @jinhojang6 maybe you could add a comment in the `src/status_im/i18n_resources.cljs` file about the `src/mocks/js_dependencies.cljs` and vice versa?

Reply:
Okay, what do you think of adding the comments to both of the files?

Reply:
Yes, that's exactly what I had in mind.
https://www.merriam-webster.com/dictionary/vice%20versa

Reply:
Oh sorry, I read your comment wrong. I read you wanted to add it to one of the files. 

No problem, will update this PR when Ricardo gives me the Portuguese translations. @jakubgs